### DISCUSSION TITLE
Improved shop blacklist

### DISCUSSION THREAD

**Discussion by WeylonSantana** [AscensionGameDev/Intersect-Engine#1532] [Ideas]
**Is your feature request related to a problem? Please describe.**
Since we can manage which items can and cannot be sold, an area of the shop editor has become useless when we switch to blacklist
![image](https://user-images.githubusercontent.com/63019821/140592790-428798c5-7cc9-49e8-8d87-8143168b8d2a.png)

**Describe the solution you'd like**
When switching to blacklist hide all items listed inside the red square above

**Additional context**
A little reminder of the blacklist function:

When we switch to the blacklist, we want the store to buy all existing items except items that we manually add there. Now we can choose which items will be sold or not in the item editor. So there is no need to add an item there in the blacklist editor when we can mark it in the item editor.
If we're in blacklist and we've added an item that we don't want the store to buy and in the item editor we put it that can sell, the item won't be bought by the store.
If we are in blacklist and left empty (buy everything), and in the item editor we put not to sell, the item will not be bought by the store.

Maybe I'm missing a few more use cases, but for comfort, I think it would be better to hide everything when switching to blacklist and leave it there only when whitelist.

Maybe there could be a label in place of everything saying "When blacklisted this store will buy all items for the price set in the item editor by the currency set in the default currency", this is already too silly, but it could be less confusing for beginners

**Comment by Daywalkrr** [comment]
Another small change I think should happen is displaying of an item's price attribute when an item is selected. Would make hopping between the shop editor and the item editor to make sure you have prices right a thing of the past.
